pinot-core/src/test/java/com/linkedin/pinot/core/predicate/NoDictionaryInPredicateEvaluatorTest.java

```diff
@@ -225,7 +225,7 @@ public void testStringPredicateEvaluators() {
     Set<String> valueSet = new HashSet<>();
 
     for (int i = 0; i < 100; i++) {
-      stringValues[i] = RandomStringUtils.random(MAX_STRING_LENGTH);
+      stringValues[i] = RandomStringUtils.random(MAX_STRING_LENGTH).replace("\t", "");
```

Comment:
Does  RandomStringUtils.random return UTF-8? If not, we may want to use a library that returns UTF-8 strings.

Reply:
Discussed offline